sympy/assumptions/assume.py

```diff
@@ -21,18 +27,34 @@ class to create your own local assumptions contexts. It is basically a thin
     Examples
     ========
 
-    >>> from sympy import Q
-    >>> from sympy.assumptions.assume import global_assumptions
+    Default assumption context is ``global_assumptions``, which is empty
+    by default.
+
+    >>> from sympy import Q, global_assumptions
     >>> global_assumptions
     AssumptionsContext()
+
+    You can add default assumption.
```

Comment:
I think this doc should really point users towards the assuming function. At least there should be a see also linking to assuming.